src/com/ikhokha/techcheck/Main.java

```diff
@@ -33,7 +33,8 @@ public static void main(String[] args) {
 	private static void addReportResults(Map<String, Integer> source, Map<String, Integer> target) {
 
 		for (Map.Entry<String, Integer> entry : source.entrySet()) {
-			target.put(entry.getKey(), entry.getValue());
+			target.putIfAbsent(entry.getKey(), 0);
+			target.put(entry.getKey(), entry.getValue()+target.get(entry.getKey()));
```

Comment:
Initialise the target `map` when empty to make sure not exception is thrown where we trying to access the current value for addition